### ISSUE TITLE
react-native-web on webOS

### ISSUE THREAD

**Issue by piotrp9898** [necolas/react-native-web#2628]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the issue

I am writing an app for webOS (LG tv) using react-native-web.  I have problem that when I launch the app there is no content from react being shown on the screen. When I change it to plain text displaying IP adress, it works and displays the adress on the screen, but when I change it to react it shows nothing but blank html page and in console there is Uncaught TypeError - root.getRootnode is not a function - I am guessing that AppRegistry,runApplication causes it. Browser on LG is Chromium 68. I even downloaded this browser on my Pc and its working on PC.

### Expected behavior

App showing two Pressable Elements and Text with IP adress

### Steps to reproduce

"dependencies": {
    "core-js": "^3.14.0",
    "react-dom": "^18.2.0",
    "react-native-web": "^0.19.10",
    "regenerator-runtime": "^0.13.7",
    "whatwg-fetch": "^3.6.2"
  },
  "devDependencies": {
    "@babel/core": "^7.23.7",
    "@babel/preset-env": "^7.23.7",
    "@babel/preset-react": "^7.23.3",
    "@webosose/ares-cli": "^2.2.0",
    "add-asset-webpack-plugin": "^2.0.1",
    "babel-loader": "^8.3.0",
    "babel-plugin-react-native-web": "^0.19.10",
    "copy-webpack-plugin": "^9.0.0",
    "metro-react-native-babel-preset": "^0.77.0",
    "webpack": "^5.75.0",
    "webpack-cli": "^4.7.1",
    "webpack-dev-server": "^4.15.1"
  }

Those are my deps.

### Test case

https://codesandbox.io/p/sandbox/52x1871vjl?file=%2Fpublic%2Findex.html

**Comment by rishabh-arch** [comment]
Hey I Need your personal help to create webOS app. Waiting for your reply

**Comment by ali-sao** [comment]
I don't believe it's a react-native-web issue. I can build my app on webos without issues
